### PR TITLE
Store id in raw_data

### PR DESCRIPTION
**Pull request by rhiaro**
Renames current data_id field in raw_data to rpde_id.
Uses data_id instead for the value of @id/id/feedUrl+identifier in the OA data object.

Heads up @michaelwood @Lathrisk @robredpath, this is a breaking change to the database schema that means you should completely delete your local copy and re-download raw data when merged.

### CODE REVIEW

File: src/lib/download-raw.js [hunk 1]
```diff
@@ -37,21 +37,24 @@ async function store_raw_callback(publisher_feed, data, nextURL) {
...
                 ];
 
                 await client.query(
-                    'INSERT INTO raw_data (publisher_feed_id, data_id, data_deleted, data_kind, data_modified, data, normalised) ' +
-                    'VALUES ($1, $2, $3, $4, $5, $6, \'f\') ' +
-                    'ON CONFLICT (publisher_feed_id, data_id) DO UPDATE SET ' +
+                    'INSERT INTO raw_data (publisher_feed_id, rpde_id, data_id, data_deleted, data_kind, data_modified, data, normalised) ' +
+                    'VALUES ($1, $2, $3, $4, $5, $6, $7, \'f\') ' +
+                    'ON CONFLICT (publisher_feed_id, rpde_id) DO UPDATE SET ' +
                     'data_deleted=$3, data_modified=$5, data=$6, updated_at=(now() at time zone \'utc\'), normalised=\'f\', validation_done=\'f\', validation_passed=\'f\', validation_results=NULL'  ,
```
Comment (odscjames) [inline src/lib/download-raw.js:58]: maybe we want data_id =$3 here just in case they do change id, even tho they shouldn't? but we can add later?

Comment (odscjames) [inline src/lib/download-raw.js:58]: Oh wait, all these numbers are wrong now anyway as you added an extra variable :-)

Comment (odscjames) [reply src/lib/download-raw.js:58]: data_deleted=$4 (i think), etc :-)

File: src/lib/web-rpde-query.js [hunk 2]
```diff
@@ -28,11 +28,11 @@ class RPDEQuery {
     sql += " WHERE updated_at < ((now() at time zone 'utc') - interval '"+ this.dont_serve_data_until_seconds_old +" seconds') ";
     // Now if user has specified after, add those
     if (this.afterTimestamp && this.afterId) {
-       sql += " AND ( ( extract(epoch from updated_at)::int = $2 AND data_id > $3) OR (extract(epoch from updated_at)::int > $2)) ";
+       sql += " AND ( ( extract(epoch from updated_at)::int = $2 AND rpde_id > $3) OR (extract(epoch from updated_at)::int > $2)) ";
```
Comment (odscjames) [inline src/lib/web-rpde-query.js:31]: All this file is the wrong table - if you look at line 25 it's normalised_data not raw_data so this shouldn't be changed

**Comment by rhiaro** [comment]
[Normalisations](https://trello.com/c/C70upiyi/10-normalisations)

**Pull request merged**